CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to this project will be documented in this file.
 
 * fix(resources): fix browser compatibility for host and os detectors [#3004](https://github.com/open-telemetry/opentelemetry-js/pull/3004) @legendecas
 * fix(sdk-trace-base): fix crash on environments without global document [#3000](https://github.com/open-telemetry/opentelemetry-js/pull/3000) @legendecas
+* fix(fetch): pass spanResponse to _endSpan() [#3037](https://github.com/open-telemetry/opentelemetry-js/pull/3037) @Ugzuzg
```

Comment:
Thanks for adding the entry. :slightly_smiling_face:  Could you move it to `experimental/CHANGELOG.md`? We keep them separate so that we can create separate releases for stable and non-stable packages.

Reply:
Oh, right. Fixed.